db/seeds.rb

```diff
@@ -5,3 +5,12 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+users = %w(macio basia seba mati brajan szefu lysy adik jan janusz)
+
+users.each_with_index do |user, index|
+  User.create(password: "password", email: "#{user}@email.com")
+end
+
+user1 = User.first
+user1.admin = true
```

Comment:
this will probably not save this changed attribute

Reply:
done